ci-operator/jobs/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master-presubmits.yaml

```diff
@@ -10,65 +10,41 @@ presubmits:
     decoration_config:
       skip_cloning: true
     labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
```

Comment:
@astoycos , do you know what this flag does?

Reply:
Not 100% sure but I would assume it just become managed by OCP's prow setup, so for example we can find it by searching here -> https://steps.ci.openshift.org/search

@dcbw  would need to confirm